Packs/Oletools/Scripts/OletoolsScript/OletoolsScript.yml

```diff
@@ -0,0 +1,144 @@
+commonfields:
+  id: Oletools
+  version: -1
+name: Oletools
+script: ''
+type: python
+args:
+- default: false
+  description: The ole command to activate. 
+              oleid - to analyze OLE files to detect specific characteristics usually found in malicious files.
+              oleobj - to extract embedded objects from OLE files.
+              olevba - to extract and analyze VBA Macro source code from MS Office documents (OLE and OpenXML).
+  isArray: false
+  name: ole_command
+  required: true
+  secret: false
+  auto: PREDEFINED
+  predefined:
+    - 'oleid'
+    - 'oleobj'
+    - 'olevba'
+- default: false
+  description: The file to activate the oletools analysis on.
+  isArray: false
+  name: entryID
+  required: true
+  secret: false
+- default: false
+  description: Display all the obfuscated strings with their decoded content (Hex, Base64, StrReverse, Dridex, VBA). Note that this works only with the olevba command.
+  isArray: false
+  name: decode
+  auto: PREDEFINED
+  predefined:
+    - true
+    - false
+  required: false
+  secret: false
+- default: false
+  description: If encrypted office files are encountered, try decryption with this password. (if specified dont specify the non_secret_password parameter)
+  isArray: false
+  name: password
+  required: false
+  secret: true
+- default: false
+  description: If encrypted office files are encountered, try decryption with this password. This argument is non secret for playbook flows. (if specified dont specify the password parameter)
+  isArray: false
+  name: non_secret_password
+  required: false
+comment: This is an automation to run oletools malware analysis for office files. Oletools is a tool for analyzing Microsoft OLE2 files, 
+  such as Microsoft Office documents or Outlook messages, mainly for malware analysis, forensics, and debugging. 
+  This automation allows performing some basic oletools commands from Cortex XSOAR.
+  Note that oletools is open source code and is subject to change.
+outputs:
```

Comment:
Please also add the DBotScore outputs: Indicator, Type, Vendor, Score.

Reply:
Doen